src/python_framework/nhd_network_utilities.py

```diff
@@ -451,10 +489,14 @@ def set_supernetwork_data(
         return dict
 
     elif supernetwork == 'CONUS_FULL_RES_v20':
+
+        ROUTELINK=r"RouteLink_NHDPLUS.nc"
+
         return {
             'geo_file_path' : os.path.join(geo_input_folder
                     , r'Channels'
-                    , r'RouteLink_NWMv2.0_20190517_cheyenne_pull.nc')
+                    , ROUTELINK)
+            , 'data_link' : f'https://www.nco.ncep.noaa.gov/pmb/codes/nwprod/nwm.v2.0.2/parm/domain/{ROUTELINK}'
```

Comment:
How do you intend to handle NWM version changes here?

Reply:
The version change is inconsequential at this point for general performance and usability testing -- we want a test data file to be available for anyone downloading the file. Anyone with a use case specific enough to require 2.1 should have access to it.

Reply:
But since it's a simple Change, I added some simple version information in the file name so that it is easier later.